rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/test_is_economizer_modeled_in_proposed.py

```diff
@@ -0,0 +1,208 @@
+                                "air_economizer": {
+                                    "id": "Air Economizer 1",
+                                    "type": "ENTHALPY",  # enthalpy economizer type
+                                },
+                            },
+                        }
+                    ],
+                }
+            ],
+        }
+    ],
+}
+
+TEST_RMD_NO_ECONOMIZER_TYPE = {
+    "id": "test_rmd",
+    "buildings": [
+        {
+            "id": "Building 1",
+            "building_open_schedule": "Required Building Schedule 1",
+            "building_segments": [
+                {
+                    "id": "Building Segment 1",
+                    "zones": [
+                        {
+                            "id": "Thermal Zone 1",
+                            "thermostat_cooling_setpoint_schedule": "Required Cooling Schedule 1",
+                            "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                            "terminals": [
+                                {
+                                    "id": "Air Terminal",
+                                    "is_supply_ducted": True,
+                                    "type": "CONSTANT_AIR_VOLUME",
+                                    "served_by_heating_ventilating_air_conditioning_system": "System 9",
+                                }
+                            ],
+                        }
+                    ],
+                    "heating_ventilating_air_conditioning_systems": [
+                        {
+                            "id": "System 9",
+                            "heating_system": {
+                                "id": "Furnace Coil 1",
+                                "heating_system_type": "FURNACE",
+                                "energy_source_type": "NATURAL_GAS",
+                            },
+                            "fan_system": {
+                                "id": "CAV Fan System 1",
+                                "fan_control": "CONSTANT",
+                                "supply_fans": [{"id": "Supply Fan 1"}],
+                                "return_fans": [{"id": "Return Fan 1"}],
+                                "air_economizer": {
+                                    "id": "Air Economizer 1",  # No economizer type
+                                },
+                            },
+                        }
+                    ],
+                }
+            ],
+        }
+    ],
+}
+
+
+TEST_RMD_FULL_FIXED_TYPE = {
+    "id": "229_01",
+    "ruleset_model_instances": [TEST_RMD_FIXED_TYPE],
+}
+
+TEST_RMD_FULL_ENTHALPY_TYPE = {
+    "id": "229_01",
+    "ruleset_model_instances": [TEST_RMD_ENTHALPY_TYPE],
+}
+
+TEST_RMD_FULL_NO_ECONOMIZER_TYPE = {
+    "id": "229_01",
+    "ruleset_model_instances": [TEST_RMD_NO_ECONOMIZER_TYPE],
+}
+
+
+def test__BASELINE_TEST_RMD__is_valid():
```

Comment:
Shouldn't the test function names reflect the names of the dictionaries being tested?  So this one would be `test__TEST_RMD_FIXED_TYPE__is_valid`. And similarly for the other test function names.